test/embind/embind_test.cpp

```diff
@@ -74,6 +75,10 @@ val emval_test_passthrough(val v) {
 void emval_test_return_void() {
 }
 
+std::nullptr_t emval_test_return_nullptr_t() {
```

Comment:
Why would a function have a return type of `std::nullptr_t`. I've seen it used for overloading with parameters but not this?